buildSrc/src/main/kotlin/Project.kt

```diff
@@ -131,7 +131,7 @@ object Versions {
   const val squareLeakCanary = "2.7"
   const val materialDesign = "1.2.0"
   const val googlePlayServicesLocation = "18.0.0"
-  const val googleCarAppLibrary= "1.1.0"
+  const val googleCarAppLibrary= "1.3.0-alpha01"
```

Comment:
```suggestion
  const val googleCarAppLibrary= "1.3.0-beta01"
```

Looks like a beta was released recently https://developer.android.com/jetpack/androidx/releases/car-app#1.3.0-beta01